agimus_controller/tests/test_warm_start.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import numpy as np
+from agimus_controller.warm_start_base import WarmStartBase
+from agimus_controller.warm_start_reference import WarmStartReference
+from agimus_controller.trajectory import TrajectoryPoint
+
+
+class TestWarmStart(unittest.TestCase):
```

Comment:
You should create on file for the base class and one file for the WarmStartReference.